pkg/ibmvpc/examples/out/analysis_out/filters_split_lb_subnet_all_vpcs__no_lbAbstract.txt

```diff
@@ -36,70 +15,91 @@ alb[LB private IP][10.240.65.4] => vsi0-sub3[10.240.128.4] : All Connections
 vsi0-ctrl-sub2[10.240.66.4] => vsi1-sub3[10.240.128.5] : All Connections
-vsi0-sub1[10.240.0.5] => alb[Fake LB private IP][10.240.1.4] : All Connections
-vsi0-sub2[10.240.64.5] => alb[Fake LB private IP][10.240.1.128] : All Connections
-vsi0-sub2[10.240.64.5] => alb[Fake LB private IP][10.240.1.4] : All Connections
-vsi0-sub2[10.240.64.5] => alb[Fake LB private IP][10.240.129.128] : All Connections
-vsi0-sub2[10.240.64.5] => alb[Fake LB private IP][10.240.65.128] : All Connections
+vsi0-sub1[10.240.0.5] => alb[Potential LB private IP][10.240.1.0/25] : All Connections
 vsi0-sub2[10.240.64.5] => alb[LB private IP][10.240.65.4] : All Connections
+vsi0-sub2[10.240.64.5] => alb[Potential LB private IP][10.240.1.0/25] : All Connections
+vsi0-sub2[10.240.64.5] => alb[Potential LB private IP][10.240.1.128/25] : All Connections
+vsi0-sub2[10.240.64.5] => alb[Potential LB private IP][10.240.129.128/25] : All Connections
+vsi0-sub2[10.240.64.5] => alb[Potential LB private IP][10.240.65.128/25] : All Connections
 vsi0-sub2[10.240.64.5] => vsi0-ctrl-sub2[10.240.66.4] : All Connections
 vsi0-sub2[10.240.64.5] => vsi0-sub3[10.240.128.4] : All Connections
 vsi0-sub2[10.240.64.5] => vsi1-sub1[10.240.0.4] : All Connections
 vsi0-sub2[10.240.64.5] => vsi1-sub2[10.240.64.4] : All Connections
 vsi0-sub2[10.240.64.5] => vsi1-sub3[10.240.128.5] : All Connections
-vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.1.128] : All Connections
-vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.1.4] : All Connections
-vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.129.128] : All Connections
-vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.65.128] : All Connections
 vsi0-sub3[10.240.128.4] => alb[LB private IP][10.240.65.4] : All Connections
+vsi0-sub3[10.240.128.4] => alb[Potential LB private IP][10.240.1.0/25] : All Connections
+vsi0-sub3[10.240.128.4] => alb[Potential LB private IP][10.240.1.128/25] : All Connections
+vsi0-sub3[10.240.128.4] => alb[Potential LB private IP][10.240.129.128/25] : All Connections
+vsi0-sub3[10.240.128.4] => alb[Potential LB private IP][10.240.65.128/25] : All Connections
 vsi0-sub3[10.240.128.4] => vsi0-ctrl-sub2[10.240.66.4] : All Connections
 vsi0-sub3[10.240.128.4] => vsi0-sub2[10.240.64.5] : All Connections
 vsi0-sub3[10.240.128.4] => vsi1-sub1[10.240.0.4] : All Connections
 vsi0-sub3[10.240.128.4] => vsi1-sub2[10.240.64.4] : All Connections
 vsi0-sub3[10.240.128.4] => vsi1-sub3[10.240.128.5] : All Connections
-vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.1.128] : All Connections
-vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.1.4] : All Connections
-vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.129.128] : All Connections
-vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.65.128] : All Connections
 vsi1-sub1[10.240.0.4] => alb[LB private IP][10.240.65.4] : All Connections
+vsi1-sub1[10.240.0.4] => alb[Potential LB private IP][10.240.1.0/25] : All Connections
```

Comment:
Here the dst pip are not the same as before, same question

Reply:
same answer